config/locales/server.en.yml

```diff
@@ -245,6 +247,7 @@ en:
         dall_e: "Generate image using DALL-E 3"
         search_meta_discourse: "Search Meta Discourse"
       command_description:
+        web_browser: "Reading <a href='%{url}'>%{url}</a>"
```

Comment:
Do we need any of those other attrs like `target="_blank"`, `rel="nofollow"` etc? Just a thought, I don't know either.

Reply:
Oh I think we get it automatically when it is cooked

Reply:
rabbit hole warning ... we cook as TL3... so everything the AI says gets follow ... I guess its a general framework fix we need. https://github.com/discourse/discourse-ai/blob/c22e5e66c50559508bee81b30d447ab526609395/lib/ai_bot/playground.rb#L268-L282 no way to tell this at the moment that we want it to nofollow and it is tied to the bot user which is tl3 :( going to track this on dev.